ddtrace/contrib/celery/util.py

```diff
@@ -0,0 +1,31 @@
+# Project
+from ddtrace import Pin
+
+# Service info
+APP = 'celery'
+SERVICE = 'celery'
+
+
+def meta_from_context(context):
+    """ helper to extract meta values from a celery context """
+    meta_keys = (
+        'called_directly', 'correlation_id', 'delivery_info', 'eta', 'expires', 'hostname',
```

Comment:
@brettlangdon exactly. all of this will be collected, stored and serialized millions of times and if nobody cares, better to be light.